src/Neo.CLI/Settings.cs

```diff
@@ -44,24 +44,30 @@ public static Settings Default
                     Initialize(config);
                 }
 
-                return _default;
+                return Custom ?? _default;
             }
         }
 
+        public static Settings? Custom { get; set; }
```

Comment:
We should wait for https://github.com/neo-project/neo/pull/3042

Reply:
@shargon

Reply:
Now we need to fix the conflicts  🤣

Reply:
🥹🥹🥹🥹🥹